### ISSUE TITLE
Cannot paste the login URL on macOS

### ISSUE THREAD

**Issue by agross** [mkb79/audible-cli#41]
I'm trying to login using an external browser. The quickstart output notes:

```
IMPORTANT:
If you are using MacOS and have trouble insert the login result url, simply import the readline module in your script.
```

This is indeed the case. I can paste the URL, but it is not accepted after pressing Enter. I tried patching `/usr/local/bin/audible` with the extra import and the URL got accepted.

**Comment by mkb79** [comment]
Thanks for the information. You can remove all key/value pairs from the URL query except the `openid.oa2.authorization_code` pair. This will work too.

**Comment by agross** [comment]
I'm currently scripting this for a non-tech-savvy friend. I think he might not understand what you suggest ;-)

**Comment by mkb79** [comment]
Maybe I should make `audible-cli` accept the pure `authorization_code` as an alternative way.

**Comment by agross** [comment]
Or import readline?

**Comment by mkb79** [comment]
`readline` is not available on all platforms. For example Windows. So that's not a good idea. But I can do this in a try/except block or check the platform before import the `readline` module. But this need some tester with Mac OS.

**Comment by agross** [comment]
I am a macOS user ;-)

**Comment by agross** [comment]
(And a Python noob for the most part. Thanks for explaining the readline issues!)

**Comment by mkb79** [comment]
No problem at all. I will implement:

```python
try:
    import readline
except ImportError:
    pass
```

This changes must be tested for side effects before they go in the master branch. Will do this next days. It's late in Germany.

**Comment by agross** [comment]
Thank you, let me know when and how I should test. (And you're right about CEST ;-)

**Comment by mkb79** [comment]
@agross Can you try the changes from dev branch please?

**Comment by agross** [comment]
Pasting works with the dev version!

**Comment by taxilian** [comment]
I just wanted to pop in and say thanks =] I came here looking to file the same issue report and the same fix, but find it already here! This is exactly what I was looking for so that I could back up all my audible books -- there are quite a number. (but I'm not addicted, I can quit any time I finish my next book...)

Granted it's anecdotal at best, but it's a good sign from an open source project =]

**Comment by mkb79** [comment]
Good to hear that it works. I published v0.0.5 to PyPI which contains all the changes.
